### PR TITLE
[12.0][ADD] stock_barcodes: Add option to get lots automatically in inventory depends on removal strategy

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT27844
ping @chienandalu @carlosdauden

### CODE REVIEW

File: stock_barcodes/wizard/stock_barcodes_read_inventory.py [hunk 1]
```diff
@@ -4,13 +4,20 @@
...
 
 
 class WizStockBarcodesReadInventory(models.TransientModel):
     _name = 'wiz.stock.barcodes.read.inventory'
     _inherit = 'wiz.stock.barcodes.read'
     _description = 'Wizard to read barcode on inventory'
 
+    def _default_auto_lot(self):
+        ICP = self.env['ir.config_parameter'].sudo()
+        auto_lot = ICP.get_param("stock_barcodes.auto_lot")
```
Comment (chienandalu) [inline stock_barcodes/wizard/stock_barcodes_read_inventory.py]: Wouldn't it be better a company setting?

Comment (sergio-teruel) [reply stock_barcodes/wizard/stock_barcodes_read_inventory.py]: Yes you are right, I have planned do it for future improvements.

**Comment by sergio-teruel** [comment]
@chienandalu changed behavior. Now the quantities have been distributed in all lots...

**Review by chienandalu — APPROVED** [review]
:+1:

**Review by carlosdauden — APPROVED** [review]

**Comment by sergio-teruel** [comment]
@carlosdauden @chienandalu Added settings view.. :face_with_head_bandage: :smile:

**Comment by chienandalu** [comment]
Great :smile:

**Review by marcelsavegnago — APPROVED** [review]

**Pull request merged**